src/Migrate/custom/Set-AzMigrateServerReplication.ps1

```diff
@@ -635,6 +635,10 @@ function Set-AzMigrateServerReplication {
                             $updateNic.TargetStaticIPAddress = $null
                         }
                         else {
+                            $isValidIpAddress = [ipaddress]::TryParse($matchingUserInputNic.TargetStaticIPAddress,[ref][ipaddress]::Loopback)
```

Comment:
PS C:\Users\prsadhu> [ipaddress]::TryParse("4",[ref][ipaddress]::Loopback)
True

Is this a valid scenario?

Reply:
Yes, it will treat it as 0.0.0.4 . Same behavior what we kept in service

Reply:
Should this be here, or in the other cmdlet where we set NIC details.

Reply:
I feel it should be in the other cmdlet.